mixer/cmd/bundles.go

```diff
@@ -278,7 +266,6 @@ func init() {
 
 	bundleListCmd.Flags().BoolVar(&bundleListFlags.tree, "tree", false, "Pretty-print the list as a tree.")
 
-	bundleEditCmd.Flags().BoolVar(&bundleEditFlags.copyOnly, "suppress-editor", false, "Suppress launching editor (only copy to local-bundles or create template)")
```

Comment:
Actually we should not remove this right now, can you just make it a no-op and also mark it as hidden, so it doesn't show up in the list anymore, but it also won't crash anyone who already is calling this in a script.

Reply:
Look for `MarkHidden()` and `MarkDeprecated()` in the code for exemples.

Reply:
@rchiossi Its already been marked hidden and deprecated. You see this line as removed because the flag has been renamed from bundleEditCmd to bundleCreateCmd.